drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java

```diff
@@ -266,6 +270,27 @@ protected void save(final String commitMessage) {
                                                                                                                  commitMessage);
     }
 
+    @Override
+    protected Command getSaveAndRename() {
```

Comment:
I think we should not override `getSaveAndRename` in this way. It differs form `BaseEditor.getSaveAndRename` just in one line and it duplicates 10 lines. I think we should move the current `ScenarioSimulationEditorBusinessCentralWrapper.getSaveAndRename` into `BaseEditor`. Then In `BaseEditor` implement `beforeSaveAndRenameCommand` as no operation and here in `ScenarioSimulationEditorBusinessCentralWrapper` override just one method `getBeforeSaveAndRenameCommand`. Please let me know what do you think.

Reply:
@jomarko I think your proposed change fits better with current code in `BaseEditor` class, then I applied it as you suggested. Thx!